modules/logging-multi-clf-enabling-additional-deployments.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-multi-clf.adoc
+
+:_content-type: PROCEDURE
+[id="logging-multi-clf-enabling-additional-deployments_{context}"]
+= Enabling multiple log forwarder mode
+
+Cluster administrators must explicitly enable multiple log forwarder mode in the cluster by configuring RBAC permissions.
+
+.Prerequisites
+
+* The Cluster Logging Operator is installed in the `openshift-logging` namespace.
+
+.Procedure
+
+. Create a service account for the collector. If you want to write logs to storage that requires a token for authentication, you must include a token in the service account.
+
+. Create cluster roles to define which log types each role is allowed to collect:
++
+.ClusterRole resource example
```

Comment:
These roles are distributed in with the operator manifest.  Maybe we wish to mention that. Also the roles are named:  `collect-[audit|application|infrastructure]-logs` which correspond to the log types we support.  It may be worth mentioning that as well

Reply:
This is great information, thank you. So basically those roles already exist and they just need to do the binding bit?

Reply:
correct.